pkg/ansible/paramconv/paramconv.go

```diff
@@ -70,6 +70,49 @@ func ToCamel(s string) string {
 	return ret
 }
 
+// This function modifies the string to handle special words that are a part of wordMapping
+/* In this function we process the input string and find the occurrence of the values in wordMapping
+   and we append "_" on both sides of that word so that it is considered to be one word*/
+// We even handle plurals of the values in wordMapping
+// If we find "s" i.e plural, we capitalize it so that it is considered to be a part of same abbreviation
+
+func preprocessWordMapping(s string) string {
```

Comment:
In go the comment should start // preprocessWordMapping ...

Following a suggestion.  (please check the example and adapt as you wish)

```suggestion
// preprocessWordMapping will check if the string informed contains special words mapped in the `wordMapping` // and its plurals and return it handled accordingly to do the snake. Note that, if the special word or its plural is 
// found the character `_` is appended to as prefixes and postfixes to the special world found. For example, if the
// string is `egressIP` the IP is a special word and then, the string   egress_ip will be returned. Also, beware that if
// the next character of the special world be an S( i.e plural ), it will be capitalized for the value be considered part
// of the same abbreviation.
func preprocessWordMapping(s string) string {
```

Reply:
ps IMO would be easier to understand if we rename s for value.